src/main/webapp/webpack.config.js

```diff
@@ -8,6 +8,9 @@ const i18nThymeleafWebpackPlugin = require("./webpack/i18nThymeleafWebpackPlugin
 const entries = require("./entries");
 const formatAntStyles = require("./styles");
 
+const SpeedMeasurePlugin = require("speed-measure-webpack-plugin");
```

Comment:
I left this in here for now, just so we can see the performance. We could remove it if we wanted to though (it is raising a deprecation warning during build).